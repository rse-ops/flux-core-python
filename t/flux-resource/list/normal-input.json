{
  "all": {
    "execution": {
      "R_lite": [
        {
          "children": {
            "core": "0-3"
          },
          "rank": "0-3"
        }
      ]
    },
    "version": 1
  },
  "allocated": {
    "execution": {
      "R_lite": []
    },
    "version": 1
  },
  "down": {
    "execution": {
      "R_lite": []
    },
    "version": 1
  }
}
