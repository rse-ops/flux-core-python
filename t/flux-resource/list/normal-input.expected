     STATE PROPERTIES NNODES   NCORES    NGPUS
      free                 4       16        0
 allocated                 0        0        0
      down                 0        0        0
